honestcash-v2/src/app/main/main-container.component.html

```diff
@@ -0,0 +1,2 @@
+<core-header></core-header>
```

Comment:
main-container makes core-header to be shared (sticky) amongs main pages (feed, settings etc is to be inside main but as sub modules)